### PR TITLE
EVP Automated Test LPS-182727, LPS-166360, LPS-166361, LPS-166359, LPS-184437|master

### PR DESCRIPTION
**Pull request by juliosilveira77**
https://issues.liferay.com/browse/LPS-182727
https://issues.liferay.com/browse/LPS-166360
https://issues.liferay.com/browse/LPS-166361
https://issues.liferay.com/browse/LPS-166359
https://issues.liferay.com/browse/LPS-184437

### CODE REVIEW

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:site-initializer-evp

**Comment by nikki-pru** [comment]
ci:test:osb-site-initializer-evp

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:osb-site-initializer-evp

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:osb-site-initializer-evp

**Review by nikki-pru — CHANGES_REQUESTED** [review]

**Comment by joelrocha68** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/paths/EVPUtils.path [hunk 1]
```diff
@@ -15,6 +15,31 @@
 	<td>//div//a[contains(@class,'btn-solid') and contains(.,'Sign In')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REQUEST_COLUMNS</td>
+	<td>//div[contains(@class,'lfr-layout-structure')]//h5[contains(@class,'component-text')][contains(text(),'${key_columns}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>KEBAB_OPTION</td>
```
[the remark this answers is not in the document]
Comment (juliosilveira77) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/paths/EVPUtils.path:24]: @nikki-pru a path existente voce esta se referindo a substituir o KEBAB_OPTION ou REQUEST_COLUMNS ?

**Comment by juliosilveira77** [comment]
Nikki, I'm sorry, I accidentally closed it, when I send the PR of the corrections I will open a new one

**Comment by juliosilveira77** [comment]
Nikki, I'm sorry, I accidentally closed it, when I send the PR of the corrections I will open a new one

**Comment by juliosilveira77** [comment]
Sorry, I closed it by mistake

**Pull request closed without merging**